Makefile

```diff
@@ -29,7 +29,7 @@ testsetup:
 	echo "running rosdistro! tests"
 
 test: testsetup
-	python setup.py nosetests
+	cd test && pytest
```

Comment:
Why is `cd test` needed where it wasn't previously?

Reply:
I did that to conform with the pattern used in other `ros-infrastructure` packages.